src/Resources/config/services/command_bus.xml

```diff
@@ -10,21 +10,21 @@
             <argument type="service" id="sylius_refund_plugin.event_bus" />
             <argument type="service" id="sylius.repository.order" />
             <argument type="service" id="Sylius\RefundPlugin\Validator\RefundUnitsCommandValidator" />
-            <tag name="messenger.message_handler" />
+            <tag name="messenger.message_handler" bus="sylius_default.bus"/>
```

Comment:
Shouldn't this be changed to 
```suggestion
            <tag name="messenger.message_handler" bus="sylius_refund_plugin.command_bus"/>
```
to keep current behavior? But if we want to migrate to `sylius_default.bus` (which a good decision IMHO), we can remove `sylius_refund_plugin.command_bus` config entirely